### ISSUE TITLE
Conflicts with AllStackable

### ISSUE THREAD

**Issue by KrisCris** [Latokike/lato-origins#18]
A player used these two mods together and the game crashed. 

`java.lang.IllegalAccessError: Update to non-static final field net.minecraft.class_1792.field_18672 attempted from a different method (setFoodComponent) than the initializer method <init> `.

AllStackable adds a new field to `Item.class` and injected a piece of code into `Item<init>` to initialize it.

```
    @Override
    public void setVanillaMaxCount(int vanillaMaxCount) {
        this.vanillaMaxCount = vanillaMaxCount;
    }

    @Inject(method = "<init>", at = @At("RETURN"))
    private void setVanillaMaxCount(Item.Settings settings, CallbackInfo ci) {
        setVanillaMaxCount(this.maxCount);
    }
```

LatoOrigins modifies `foodComponent` using an accessor during the item registering, which is after the first value assignment.

I don't know why these would ever conflict.

https://github.com/KrisCris/AllStackable/issues/14

**Comment by latokike** [comment]
I am currently working on a fix for this by changing the way the origins eat.

**Comment by latokike** [comment]
This has been fixed with [`6bcd5a4`](https://github.com/Latokike/lato-origins/commit/6bcd5a4202c6cb0107e040579d3f5e2b37df099b)
